buildResetExtension.js

```diff
@@ -0,0 +1,28 @@
+const { spawnSync } = require('child_process')
+const { existsSync } = require('fs')
+const { platform } = require('os')
+const process = require('process')
+
+const [nodeExe, scriptPath, ...args] = process.argv
+
+const resetExtensionScript =
+  './node_modules/@hyperplay/extension-importer/src/resetExtension.js'
+if (existsSync(resetExtensionScript)) {
+  console.log('spawning pkg', resetExtensionScript, args)
```

Comment:
Should we keep this?

Reply:
It's only logged in the build step in the github action so I think it could be useful for debugging